src/dahua.ts

```diff
@@ -36,14 +36,14 @@ class DahuaEvents {
             password: pass
         }
 
-        let keepAliveAgent
+        let keepAliveAgent: HttpAgent | HttpsAgent;

```

Comment:
Actually thinking out loud, couldn't we define both `httpAgent` and `httpsAgent` always? https://axios-http.com/docs/req_config#:~:text=//%20%60httpAgent%60%20and%20%60httpsAgent%60%20define%20a%20custom%20agent%20to%20be%20used%20when,https.Agent(%7B%20keepAlive%3A%20true%20%7D)%2C

Reply:
Makes sense to me since different cameras can use http or TLS respectively. Additionally both variables can be defined as consts, which will the resolve concern in this comment https://github.com/kushagharahi/homebridge-dahua-alerts/pull/45?notification_referrer_id=NT_kwDOADLAMrI0Nzk4MDY2Mzk4OjMzMjYwMDI#discussion_r1036861282

Reply:
Done